dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/PermissionsWrapper.java

```diff
@@ -116,6 +118,13 @@ public boolean canEditDataverseTextMessagesAndBanners(Long dataverseId) {
         return permissionService.isUserCanEditDataverseTextMessagesAndBanners(dataverseId);
     }
 
+    public boolean canEditDatasetInReview(Dataset dataset) {
```

Comment:
I think this method should be moved to PermissionServiceBean, since PermissionsWrapper is meant for view.

Reply:
As discussed brake it down to 2 booleans.